docs/en/userinfo_endpoint.rst

```diff
@@ -28,27 +34,32 @@ The UserInfo Endpoint MUST only support the method HTTP GET :rfc:`2616` and MUST
 Response
 ++++++++
 
-The response of the UserInfo Endpoint MUST specify the value "application/jwt" in the "Content-Type".
+The response of the UserInfo Endpoint MUST specify the value "application/jwt" or "application/jose" in the "Content-Type".
 
 The content of the Response body MUST be a `signed and encrypted JWT. <https://openid.net/specs/openid-connect-core-1_0.html#UserInfoResponse>`_
 
-The UserInfo Endpoint returns the authorized claims in the Authentication Request.
+The JOSE header MUST contain the *cty* parameter (Content Type) with the value *JWT* (see :rfc:`7515#section-4`).
+
 
-**Example:**
 
-.. code-block:: 
+The UserInfo Endpoint returns user attributes explicitly requested through the **claims** parameter or through the use of the **scope** parameter in the Authentication Request.
 
- {
-     "iss":"https://op.fornitore_identita.it",
-     "aud":"https://rp.fornitore_servizio.it",
-     "iat":1519032969,
-     "nbf":1519032969,
-     "exp":1519033149,
-     "sub":"OP-1234567890",
-     "name":"Mario",
-     "https://attributes.spid.gov.it/familyName":"Rossi",
-     "https://attributes.spid.gov.it/fiscalNumber":"MROXXXXXXXXXXXXX"
- }
+.. TODO: Move examples in the specific section
+  **Example:**
+
+  .. code-block:: 
```

Comment:
@fmarino-ipzs sullo sphinx in produzione in docs italia dobbiamo perforza dire anche il tipo di lexer, in questo caso ti prego di modifica tutti i `.. code-block:: ` come ad esempio `.. code-block:: json`